src/RA_Defs.cpp

```diff
@@ -4,59 +4,79 @@
 #include <Windows.h>
 #include <locale>
 #include <codecvt>
```

Comment:
`locale` and `codecvt` probably aren't used any more.

Reply:
Probably not